AI/StupidAI/StupidAI.cpp

```diff
@@ -94,20 +96,23 @@ bool isCloser(const EnemyInfo & ei1, const EnemyInfo & ei2, const ReachabilityIn
 
 }
 
-static bool willSecondHexBlockMoreEnemyShooters(const BattleHex &h1, const BattleHex &h2)
+static bool willSecondHexBlockMoreEnemyShooters(const BattleHex & h1, const BattleHex & h2)
 {
-	int shooters[2] = {0}; //count of shooters on hexes
+	int shooters[2] =
+	{
+		0

```

Comment:
@alexvins because if you initialize less elements than exist, rest is zero initialized. This syntax can initialize array of any size with zeros.